guides/doc-Upgrading_and_Updating/topics/con_upgrading-prerequisites.adoc

```diff
@@ -0,0 +1,28 @@
+[[upgrading_prerequisites]]
```

Comment:
IMO, the `prerequisite` does not require a separate file. It can be covered in the `upgrading_satellite_server`

Reply:
I have kept the prerequisites as a separate chapter, as it makes it contains some important pre-upgrade information. I feel keeping it under `upgrading_satellite_server`, might affect its visibility. What do you think?

Reply:
Owkay. This perspective also makes sense. We can go with a separate section :+1: